book/http.md

```diff
@@ -583,13 +583,6 @@ not recommended. It's very difficult to write a custom TLS
 implementation that is not only correct but secure.
 :::
 
-A reference implementation
-==========================
-
-There are of course many implementations already of the HTTP protocol. You
-should take advantage of this to use one of the existing implementations
-as a *reference implementation*, in order to test the whether your 
```

Comment:
Oops. this was meant to be finished off with statements like comparing your implementation with curl. Let's just delete it for now.